dopex-io/elvarg
# fix(dopex-ui): menu component

## Scope

Use headlessui/listbox instead of headlessui/menu to be more in-line with the intended behavior of Dopex menus.  

## Implementation

Menu/MenuItems was fully scrapped since its event handlers were not flexible enough to update object-based states. Our Menu component now uses Headless UI's Listbox, which allows us to update React states more easily. 

## Screenshots

No changes in styling.

## How to Test

Visit the Beta Ssov page to see if any unintended behaviors arise from the market selection Menu. This is the only place currently using dopex-ui/Menu
